### PR TITLE
Clarify minor version skew

### PR DESCRIPTION
**Pull request by jamiehannaford**
/cc @luxas

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5819)

### CODE REVIEW

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -506,6 +506,9 @@ Due to that we can't see into the future, kubeadm CLI vX.Y may or may not be abl
 Example: kubeadm v1.8 can deploy both v1.7 and v1.8 clusters and upgrade v1.7 kubeadm-created clusters to
 v1.8.
 
+Please also check our [installation guide](/docs/setup/independent/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)
+for more information on the version skew between kubelets and the control panel.
+
```
Comment (steveperry-53) [inline docs/setup/independent/create-cluster-kubeadm.md]: Is "control panel" correct? Or is it "control plane"?

Comment (xiangpengzhao) [reply docs/setup/independent/create-cluster-kubeadm.md]: Seems like it should be `control plane`.
BTW, sorry for the incorrect review comment in https://github.com/kubernetes/kubernetes.github.io/pull/5815#discussion_r143420140 (`s/pabel/panel`)

Comment (jamiehannaford) [reply docs/setup/independent/create-cluster-kubeadm.md]: Good catch! I just corrected

File: docs/setup/independent/install-kubeadm.md [hunk 2]
```diff
@@ -135,7 +135,9 @@ You will install these packages on all of your machines:
 kubeadm **will not** install or manage `kubelet` or `kubectl` for you, so you will 
 need to ensure they match the version of the Kubernetes control panel you want 
 kubeadm to install for you. If you do not, there is a risk of a version skew occurring that
-can lead to unexpected, buggy behaviour.
+can lead to unexpected, buggy behaviour. However, _one_ minor version skew between the 
+kubelet and the control plane is supported. For example, kubelets running 1.7.0 should be 
+fully compatible with a 1.8.0 API server.
```
Comment (luxas) [inline docs/setup/independent/install-kubeadm.md]: ...but the kubelet version may never exceed the API server version.

Comment (jamiehannaford) [reply docs/setup/independent/install-kubeadm.md]: I agree about the location split. How about we put all the version skew in the main section (where it currently lives) and link off to it after `unexpected, buggy behaviour`.

**Comment by steveperry-53** [comment]
@jamiehannaford There's still one instance of "control panel" in install-kubeadm.md.

**Comment by steveperry-53** [comment]
@jamiehannaford @luxas This seems very close to being ready. Can you make the last few changes and give a tech approval? Thanks.

**Comment by steveperry-53** [comment]
Ping @jamiehannaford

**Pull request merged**